src/client.rs

```diff
@@ -316,20 +316,18 @@ impl Spotify {
     ///
     /// Parameters:
     /// - artist_id - the artist ID, URI or URL
-    /// - country - limit the response to one particular country.
+    /// - market - limit the response to one particular country.
     ///
     /// [Reference](https://developer.spotify.com/web-api/get-artists-top-tracks/)
     #[maybe_async]
-    pub async fn artist_top_tracks<T: Into<Option<Country>>>(
+    pub async fn artist_top_tracks<T: Into<Market>>(
```

Comment:
You can remove this `Into<Market>` and just use `Market`, as `Market` doesn't implement methods like `From<String>` AFAIK. This `Into` was used previously to avoid `Some` when passing optional parameters, but this decision is being discussed at #134 anyway (you can take a look and leave your opinion if you like :))